nixos/modules/services/x11/window-managers/dk.nix

```
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.dk;
+in
+
+{
+  options = {
+    services.xserver.windowManager.dk = {
+      enable = mkEnableOption (lib.mdDoc "dk");
+
+      package = mkOption {
+        type        = types.package;
+        default     = pkgs.dk;
+        defaultText = literalExpression "pkgs.dk";
+        description = lib.mdDoc ''
+          dk package to use.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    services.xserver.windowManager.session = singleton {
+      name  = "dk";
+      start = ''
```

```suggestion
      name = "dk";
      start = ''
```

No need for such pretty spacing.